Content.Server/Shuttles/Commands/FTLDiskBurnerCommand.cs

```diff
@@ -0,0 +1,176 @@
+                    shell.WriteLine(destinations + " is somehow on map " + dest + " with no map component. What the fuck.");
+                    continue;
+                }
+                if (mapComp.MapInitialized == false)
+                {
+                    shell.WriteLine(
+                        destinations +
+                        " is on map " +
+                        dest +
+                        " which is not initialized! Check it's safe to initialize, then initialize it first or the players will be stuck in place!"
+                        );
+                    continue;
+                }
+                if (mapComp.MapPaused == true)
+                {
+                    shell.WriteLine(
+                        destinations +
+                        " is on map " +
+                        dest +
+                        " which is paused! Please unpause the map first or the players will be stuck in place."
+                        );
+                    continue;
+                }
+
+                // check if our destination works already, if not, make it.
+                if (!_entManager.TryGetComponent<FTLDestinationComponent>(dest, out var ftlDestComp))
+                {
+                    FTLDestinationComponent ftlDest = _entManager.AddComponent<FTLDestinationComponent>(dest);
+                    ftlDest.RequireCoordinateDisk = true;
+
+                    if (_entManager.HasComponent<MapGridComponent>(dest))
+                    {
+                        ftlDest.BeaconsOnly = true;
+
+                        shell.WriteLine(
+                            destinations +
+                            " is a planet map " +
+                            dest +
+                            " and will require an FTL point. It may already exist."
+                            );
+                    }
+                }
+                else
+                {
+                    // we don't do these automatically, since it isn't clear what the correct resolution is. Instead we provide feedback to the user and carry on like they know what theyre doing.
+                    if (ftlDestComp.Enabled == false)
+                        shell.WriteLine(
+                            destinations +
+                            " is on map " +
+                            dest +
+                            " that already has an FTLDestinationComponent, but it is not Enabled! Set this manually for safety.");
+
+                    if (ftlDestComp.BeaconsOnly == true)
+                        shell.WriteLine(
+                            destinations +
+                            " is on map " +
+                            dest +
+                            " that requires a FTL point to travel to! It may already exist."
+                            );
+                }
+
+                // create the FTL disk
+                EntityUid cdUid = _entManager.SpawnEntity(CoordinatesDisk, coords);
```

Comment:
I would prefer if the disk ended up the entity's hand if they have one. Otherwise an admin might end up spawning the disk on the floor unintentionally.

I think you can look at stuff like the Uplink for the correct function for that.

Reply:
I can make it try. The command makes an arbitrary number of disks so ymmv.